styler.css

```diff
@@ -0,0 +1,815 @@
+  }
+
+  .deals-sec > article {
+    position: relative;
+  }
+
+  .sec-1 > article:nth-child(2) {
+    grid-row: 1 / span 2;
+  }
+
+  .sec-1 > article > h2 {
+    font-weight: bold;
+    font-family: sans-serif;
+  }
+
+  .sec-1 > article:nth-child(2) > h2 {
+    font-size: 2em;
+  }
+
+  .sec-1 > article:not(:nth-child(2)) > h2 {
+    font-size: 1.1em;
+  }
+
+  .sec-2 {
+    grid-template-columns: repeat(3, 1fr);
+    gap: 20px;
+  }
+
+  .sec-2 .article-headings {
+    grid-column: 1 / -1;
+  }
+
+  .img {
+    width: 100%;
+    height: 18vw;
+  }
+
+  .img-description {
+    padding: 10px 0 0 0;
+  }
+
+  .aside-div {
+    flex-direction: unset;
+    justify-content: space-around;
+    padding: 0;
+  }
+
+  .links a {
+    margin-right: 15px;
+    padding: 0 15px 0 0;
+    font-size: 1.1em;
+  }
+
+  .icons-div > a {
+    margin-right: 14px;
+  }
+
+  .aside-div a:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  .links {
+    flex-direction: row;
+  }
+
+  .links a:not(:last-child) {
+    border-right: 1px solid #f42;
+  }
+
+  a.hidden-flex {
+    display: none;
+  }
+
+  .sec-4 {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    margin-right: 0;
+    padding-right: 0;
+  }
+
+  .sec-4-articles {
+    max-width: 48%;
+    margin-right: 2%;
+  }
+
+  .sec-4-articles > .main-articles {
+    height: 35vw;
+  }
+
+  .deals-div {
+    grid-column: 1 / -1;
+  }
+
+  .discount {
+    display: inline;
+    position: absolute;
+    top: 22px;
+    right: 3px;
+    color: black;
+    transform: rotate(45deg);
+    z-index: 1;
+    font-size: 0.8em;
+    font-weight: 800;
+    font-family: sans-serif;
+  }
+
+  .deals-sec > article::after {
+    position: absolute;
+    top: 0;
+    right: 0;
+    content: "";
+    border-style: solid;
+    border-width: 40px;
+    border-color: #fc0 #fc0 transparent transparent;
+  }
+
+  .lower-footer > p {
+    width: 70%;
+  }
+}
+
+@media only screen and (min-width: 1024px) {
+  .nav-logo-div {
+    display: none;
+  }
+
+  .img-description > p {
+    font-size: 1.1em;
+    font-weight: bolder;
+  }
+
+  header > p:last-child {
+    padding: 10px 0 10px 28px;
+  }
+
+  .sec-1 {
+    grid-auto-rows: 16em;
+  }
+
+  .sec-1 > article:nth-child(2) > h2 {
+    font-size: 3em;
+    font-weight: bold;
+    font-family: sans-serif;
+  }
+
+  .sec-1 > article:not(:nth-child(2)) > h2 {
+    font-size: 1.45em;
+  }
+
+  .sec-2 {
+    grid-template-columns: repeat(4, 1fr);
+  }
+
+  .img {
+    height: 13vw;
+  }
+
+  .funding-head {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-direction: row;
+  }
+
+  .links a {
+    font-size: 1em;
+  }
+
+  a.hidden-flex {
+    display: flex;
+  }
+
+  .sec-4-articles {
+    max-width: 31.5%;
+    margin-right: 1.5%;
+  }
+
+  .sec-4-articles > .main-articles {
+    height: 23vw;
+  }
+}
+
+@media only screen and (min-width: 1200px) {
+  main {
+    width: 92%;
+    margin: 0 auto;
+  }
+}
```

Comment:
This can best be achieved with the `max-width` property on your main tag. This media query can be avoided so that you have only 2 breakpoints.